exetera/core/fields.py

```diff
@@ -479,9 +479,13 @@ def __len__(self):
     @property
     def dtype(self):
         """
-        Get datatype of field.
+        Get datatype of field. Please note constructing a numpy array from IndexedString data can be very memory expensive.
```

Comment:
Get the numpy compatible datatype of the indexed string field. Note that the type that is returned is 'S{n}' where n is the longest string present in the array, as numpy can only natively handle fixed length string formats. This can be much more expensive than the indexed string field representation and so should be used with care.